pdl-compiler/scripts/generate_python_backend.py

```diff
@@ -1047,7 +1064,10 @@ def generate_packet_declaration(packet: ast.Declaration) -> str:
         elif isinstance(f, ast.ScalarField):
             field_decls.append(f"{f.id}: int = field(kw_only=True, default=0)")
         elif isinstance(f, ast.TypedefField):
-            if isinstance(f.type, ast.EnumDeclaration):
+            if isinstance(f.type, ast.EnumDeclaration) and f.type.tags[0].range:
+                field_decls.append(
+                    f"{f.id}: {f.type_id} = field(kw_only=True, default={f.type.tags[0].range[0]})")
```

Comment:
Is there a test covering this? This seems like something that should have been handled previously as well (with `{f.type.tags[0].id}_START` as default value)

Reply:
Added canonical tests for enum declarations